test-requirements.txt

```diff
@@ -13,7 +13,7 @@ coverage-config-reload-plugin~=0.2
 codecov~=2.0.5
 freezegun~=0.3.9
 moban~=0.3.3 ; python_version > '3.0'
-pytest~=3.6.1
+pytest~=4.4.0
```

Comment:
this is wrong.  you are proposing to change this in coala-mobans repo, to 3.6.4 , so that is the version that needs to be here in your PR , so we can see that 3.6.4 is OK.